ubuntu22.04/precompiled/nvidia-driver

```diff
@@ -234,15 +234,11 @@ _unload_driver() {
 
 # Link and install the kernel modules from a precompiled packages
 _install_driver() {
-    # Install necessary userspace, fabric manager and libnvidia-nscq packages
-    apt-get install -y --no-install-recommends nvidia-driver-${DRIVER_BRANCH}-server
-
-    # Uninstall unnecessary packages installed as a part of the nvidia-driver-${DRIVER_BRANCH}-server package
-    apt-get purge -y \
-        libnvidia-egl-wayland1 \
-        nvidia-dkms-${DRIVER_BRANCH}-server \
-        nvidia-kernel-source-${DRIVER_BRANCH}-server \
-        xserver-xorg-video-nvidia-${DRIVER_BRANCH}-server
+    # Install necessary driver userspace packages
+    apt-get install -y --no-install-recommends \
```

Comment:
These will just install utils like `nvidia-smi, nvidia-persistenced, nvidia-mps-control` etc, but not other required libs like encoder/decoder/gl/cuda libs etc. There are ton of other user space packages that we need. where are those installed as dependencies?

Reply:
If the `encoder/decoder/gl/cuda libs` are required, I can install them. I wasn't aware that they were needed. 

I validated the current container with mps and tests like gpu-burn. They worked fine

Reply:
I've added the other packages. Please check again